### PR TITLE
feat(#2437): data objects can be used directly

### PR DESCRIPTION
**Pull request by maxonfjvipon**
Ref: #2437

---

### Focus of the PR
This PR focuses on making several changes to the codebase related to parsing and transforming XML files.

### Detailed summary
- Removed the file "/org/eolang/parser/errors/data-objects.xsl" from the "ParsingTrain.java" file.
- Added the file "/org/eolang/parser/explicit-data.xsl" to the "ParsingTrain.java" file.
- Modified the "locators-to-java.yaml" file by removing "/org/eolang/parser/wrap-method-calls.xsl" and "/org/eolang/parser/set-locators.xsl", and adding "/org/eolang/parser/explicit-data.xsl".
- Added new content to the "eo-parser/src/test/resources/org/eolang/parser/packs/explicit-data.yaml" file.
- Added a new XML file "/org/eolang/parser/explicit-data.xsl" with specific content.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by maxonfjvipon** [comment]
@Graur please review this one

File: eo-runtime/src/test/java/EOorg/EOeolang/EOintTest.java [hunk 1]
```diff
@@ -40,7 +40,6 @@
  * @checkstyle TypeNameCheck (4 lines)
  */
 public class EOintTest {
-
```
Comment (Graur) [inline eo-runtime/src/test/java/EOorg/EOeolang/EOintTest.java:43]: @maxonfjvipon Is this change really necessary?

**Review by Graur — CHANGES_REQUESTED** [review]
@maxonfjvipon Good one, just a one question from my side:

File: eo-runtime/src/test/java/EOorg/EOeolang/EOintTest.java (hunk 1 above)
Comment (maxonfjvipon) [reply eo-runtime/src/test/java/EOorg/EOeolang/EOintTest.java:43]: @Graur no, got there by accident. Should I remove it?

Comment (Graur) [reply eo-runtime/src/test/java/EOorg/EOeolang/EOintTest.java:43]: @maxonfjvipon Yes, It would be great!

**Comment by maxonfjvipon** [comment]
@Graur please have a look one more time

**Review by Graur — APPROVED** [review]

**Comment by maxonfjvipon** [comment]
@yegor256 please have a look

**Comment by yegor256** [comment]
@maxonfjvipon can you add this test too:

```
QQ.float 3.14 > x
```

**Comment by maxonfjvipon** [comment]
@yegor256 please have a look one more time

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**